src/hooks/useReportIDs.tsx

```diff
@@ -133,7 +133,8 @@ function ReportIDsContextProvider({
             SidebarUtils.getOrderedReportIDs(
                 currentReportID ?? null,
                 chatReports,
-                betas,
+                // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
+                betas as OnyxEntry<OnyxTypes.Beta[]>,
```

Comment:
```suggestion
                betas,
```

We can remove this now since we removed `Readonly` from `useOnyx`

Reply:
thanks for the catch!